### DISCUSSION TITLE
Code for generating atmosphere/land/aerosol/wave ICs?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#2233] [Q&A] [answered]
Hello,

Could someone point me to the code and documentation for generating ICs at various resolutions for the atmosphere, aerosol ICs from MERRA2, NOAHMP, waves, etc? Thanks!

**Comment by jkbk2004** [comment]
@benjamin-cash There are some other feedbacks to follow the exactly same request. We are scheduling meetings to produce a quick start document. Starting points will be atmosphere- ICs first. I am not sure about ICs from MERRA2. We will put the items in the discussion, IC generation approaches for marine and wave as well. Will keep you posted.

**Reply by benjamin-cash** [reply]
Thanks @jkbk2004 - my understanding is there is code out there to generate aerosol ICs from MERRA2. Maybe someone from the aerosol team could comment? I would tag someone here but I'm not sure who the best person would be.

**Reply by gspetro-NOAA** [reply]
Hi @benjamin-cash, 

It sounds like @jkbk2004 has a meeting with @GeorgeGayno-NOAA to discuss some of this tomorrow, and hopefully one or both of them will have more details for you at that point. 

Best,
Gillian Petro

**Reply by benjamin-cash** [reply]
Hi @gspetro-NOAA - Thanks for the update! Looking forward to hearing the results.

**Comment by jkbk2004** [comment]
Still need to check for GOCART/MERRA2 and NOAH-MP sides. A few comments regarding FV3 and marine:
* Quick steps to generate IC for FV3 is to follow the gdas-init option in ufs-utils: https://noaa-emcufs-utils.readthedocs.io/en/latest/ufs_utils.html#gdas-init: it builds and runs ok on hera.
* Marine IC file generation bash scripts (based on CPC feedback): /gpfs/f5/epic/scratch/Jong.Kim/EMC_ugcs_C5/workflow/01.prepare_ics.sh and  /gpfs/f5/epic/scratch/Jong.Kim/FCST_initC5/3prepare_ufs_ocnice_ics.sh, no formal document about the script but straightforward to read thru the script and document,  note that some issues with CICE dynamics fields at https://github.com/ufs-community/UFS_UTILS/issues/929

**Comment by gspetro-NOAA** [comment]
It sounds from @barlage that the standard cold start files generated by `chgres_cube` have all the necessary input states to run the land model within the atmosphere.

**Comment by jkbk2004** [comment]
@benjamin-cash @gspetro-NOAA An update for marine ICs: https://github.com/ufs-community/UFS_UTILS/pull/942

**Comment by DeniseWorthen** [comment]
The documentation for the new ocnice_prep utility is https://ufs-community.github.io/UFS_UTILS/ocnice_prep/index.html

**Comment by benjamin-cash** [comment]
@NeilBarton-NOAA @pjpegion - Are the scripts you pointed me to using the various options above?

**Reply by NeilBarton-NOAA** [reply, marked as answer]
to my understanding, yes. The aerosols initial conditions are from the MERRA2 analysis. All scripts required a base ICs first.

**Reply by benjamin-cash** [reply]
Thanks! To complete the answer, I pulled down scripts from Neil's repo here : https://github.com/NeilBarton-NOAA/REPLAY_ICS
